src/me/ryanhamshire/PopulationDensity/ScanRegionTask.java

```diff
@@ -22,44 +22,158 @@
+        Material.BIRCH_LEAVES,
+        Material.JUNGLE_LEAVES,
+        Material.ACACIA_LEAVES,
+        Material.DARK_OAK_LEAVES,
+        Material.CHERRY_LEAVES,
+        Material.MANGROVE_LEAVES,
+        Material.MANGROVE_ROOTS,
+        Material.TALL_GRASS,
+        Material.BLUE_ORCHID,
+        Material.AZURE_BLUET,
+        Material.RED_TULIP,
+        Material.ORANGE_TULIP,
+        Material.PINK_TULIP,
+        Material.WHITE_TULIP,
+        Material.OXEYE_DAISY,
+        Material.SAND,
+        Material.SUSPICIOUS_SAND,
+        Material.SANDSTONE,
+        Material.RED_SANDSTONE,
+        Material.SNOW,
+        Material.VINE,
+        Material.LILY_PAD,
+        Material.DANDELION,
+        Material.MOSSY_COBBLESTONE,
+        Material.CLAY,
+        Material.SUGAR_CANE,
+        Material.PACKED_ICE,
+        Material.BLUE_ICE,
+        Material.ROSE_BUSH,
+        Material.LILAC,
+        Material.LARGE_FERN,
+        Material.GRASS_BLOCK,
+        Material.WHITE_TERRACOTTA,
+        Material.ORANGE_TERRACOTTA,
+        Material.MAGENTA_TERRACOTTA,
+        Material.LIGHT_BLUE_TERRACOTTA,
+        Material.YELLOW_TERRACOTTA,
+        Material.LIME_TERRACOTTA,
+        Material.PINK_TERRACOTTA,
+        Material.GRAY_TERRACOTTA,
+        Material.LIGHT_GRAY_TERRACOTTA,
+        Material.CYAN_TERRACOTTA,
+        Material.PURPLE_TERRACOTTA,
+        Material.BLUE_TERRACOTTA,
+        Material.BROWN_TERRACOTTA,
+        Material.GREEN_TERRACOTTA,
+        Material.RED_TERRACOTTA,
+        Material.BLACK_TERRACOTTA,
+        Material.TERRACOTTA,
+        Material.GRANITE,
+        Material.DIORITE,
+        Material.ANDESITE,
+        Material.COARSE_DIRT,
+        Material.ROOTED_DIRT,
+        Material.MOSS_BLOCK,
+        Material.MOSS_CARPET,
+        Material.MUD,
+        Material.PODZOL,
+        Material.BEDROCK,
+        Material.PEONY,
+        Material.SEAGRASS,
+        Material.TALL_SEAGRASS,
+        Material.SEA_PICKLE,
+        Material.TUBE_CORAL,
+        Material.BRAIN_CORAL,
+        Material.BUBBLE_CORAL,
+        Material.FIRE_CORAL,
+        Material.HORN_CORAL,
+        Material.TUBE_CORAL_BLOCK,
+        Material.BRAIN_CORAL_BLOCK,
+        Material.BUBBLE_CORAL_BLOCK,
+        Material.FIRE_CORAL_BLOCK,
+        Material.HORN_CORAL_BLOCK,
+        Material.TUBE_CORAL_FAN,
+        Material.BRAIN_CORAL_FAN,
+        Material.BUBBLE_CORAL_FAN,
+        Material.FIRE_CORAL_FAN,
+        Material.HORN_CORAL_FAN,
+        Material.BEE_NEST,
+        Material.SCULK,
+        Material.SCULK_CATALYST,
+        Material.SCULK_SENSOR,
+        Material.SCULK_SHRIEKER,
+        Material.SCULK_VEIN,
+        Material.GLOW_LICHEN,
+        Material.KELP
+    ));  
+
+    public ScanRegionTask(ChunkSnapshot[][] chunks, boolean openNewRegions) {
         this.chunks = chunks;
         this.openNewRegions = openNewRegions;
     }
 
-    private class Position
-    {
+    private class Position {
```

Comment:
I prefer newline braces, but, is easier if style changes are left for a separate PR.

Reply:
I will try to remove back all styling changes in another commit.